apps/dsp-app/cypress/support/pages/project-page.ts

```diff
@@ -0,0 +1,34 @@
+import { ProjectADM, ProjectOperationResponseADM } from '../../../../../libs/vre/open-api/src';
+
+class ProjectPage {
+  private payload = {
```

Comment:
payload is only used in request Project, add it as a constant of requestProject()